Lombiq.Hosting.Tenants.Admin.Login/Extensions/ActionContentExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.AspNetCore.Mvc;
+using OrchardCore.Mvc.Core.Utilities;
+using OrchardCore.Tenants.Controllers;
+
+namespace Lombiq.Hosting.Tenants.Admin.Login.Extensions;
+
+public static class ActionContentExtensions
```

Comment:
And then further changes to this internally used method won't necessitate suppressions either.

```suggestion
internal static class ActionContentExtensions
```